ios/Surveys/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  SurveyDetailView.swift
+//  Surveys
+//
+//  Created by Su Ho on 22/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SurveyDetailView: View {
+
+    @ObservedObject var viewModel: SurveyDetailViewModel
+    @EnvironmentObject private var navigator: Navigator
+
+    @State private var backgroundScale = 1.0
+
+    var body: some View {
+        ZStack {
+            GeometryReader { proxy in
+                URLImage.urlString(viewModel.imageURLString)
+                    .resizable()
+                    .scaledToFill()
+                    .scaleEffect(backgroundScale, anchor: .topTrailing)
+                    .frame(width: proxy.size.width, height: proxy.size.height)
+                    .clipped()
+                    .animation(.linear(duration: 1), value: backgroundScale)
+                    .accessibilityIdentifier(AccessibilityIdentifier.SurveyDetail.image)
+            }
+            .ignoresSafeArea()
+
+            Rectangle()
+                .foregroundColor(.clear)
+                .background(
+                    LinearGradient(colors: [.clear, .black], startPoint: .top, endPoint: .bottom)
+                )
+                .opacity(0.6)
+                .ignoresSafeArea()
```

Comment:
Should we extract this logic into a reusable view? I see we're using the same UI logic to show the background with a vertical linear gradient from [HomeSurveysView.swift](https://github.com/suho/kmm-ic/pull/127/files#diff-f92f1eacf0e3a425f4a3d60206c3a9d1e4c5ea4f4ac250341a8b41da2fa58330)

Reply:
@luongvo After @nkhanh44 [comment](https://github.com/suho/kmm-ic/pull/127#pullrequestreview-1235294390), I prefer not to extract the whole background view (include the image), but will extract for the gradient view only :D

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/127/commits/a775351590ca92cbc7eb82edde34450560b43d11